src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -97,11 +97,13 @@ public void ActiveSolutionBoundTracker_ArgChecks()
         {
             // Arrange
             Exceptions.Expect<ArgumentNullException>(() =>
-                new ActiveSolutionBoundTracker(null, new ConfigurableActiveSolutionTracker(), loggerMock.Object));
+                new ActiveSolutionBoundTracker(null, new ConfigurableActiveSolutionTracker(), loggerMock.Object, Mock.Of<IGitEvents>()));
```

Comment:
I would delete this test. As explained in another comment, it doesn't make sense for most MEF constructors.

Reply:
Updated: it's done in the Git monitor PR.